store/store.go

```diff
@@ -561,6 +575,10 @@ func (s *Store) newRequest(reqOptions *requestOptions, user *auth.UserState) (*h
 	req.Header.Set("X-Ubuntu-Series", s.series)
 	req.Header.Set("X-Ubuntu-Wire-Protocol", UbuntuCoreWireProtocol)
 
+	if os.Getenv("SNAPPY_DELTA_DOWNLOAD_FORMATS") != "" {
```

Comment:
@niemeyer Yes, I agree that eventually the env var will not be necessary, but right now it's being used as a switch as well... Ah - which is probably what you mean, I should have a boolean env var as a switch and instead add xdeltas to the internal configuration? I'll do that too :)